docs/docs/guides/frameworks/creating-a-framework-integration.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Creating a framework integration
+---
+
+The core functionalities of Auth.js - `@auth/core` - are built on top of the Web Standard [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request)/[Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) mental model, and therefore are framework-agnostic. For each framework, we provide an integration layer that allows you to use the authentication features in a way that is specific to the framework. See the [list of integrations](./reference#integrations) that are currently available.
+
+We welcome contributions of new official integrations. If you are interested in creating & maintaining a new integration, please read the following guidelines.
+
+## Official framework guidelines
+
+If you want to create a new official framework integration and distribute it under `@auth/` namespace, please make sure to follow the next steps and fulfill our maintenance requirements below:
+
+### Setting up a new integration
+
+We provide a script that generates all the required files for a new integration. To run the script, run the following command:
+
+```bash
+pnpm setup-fw-integration <framework-name>
+```
+
+This will copy all the files from our official template at `./packages/frameworks-template` to new directory under `packages/` with all the required files, and rename the placeholders to the name of the framework you provided.
```

Comment:
```suggestion
This will copy all the files from our official template at `./packages/frameworks-template` to a new directory under `packages/` with all the required files, and rename the placeholders to the name of the framework you provided.
```